packages/web-app-files/src/views/spaces/GenericSpace.vue

```diff
@@ -301,6 +306,10 @@ export default defineComponent({
 
     displayThumbnails() {
       return !this.configuration?.options?.disablePreviews
+    },
+
+    isSpaceFrontpage() {
+      return isProjectSpaceResource(this.space) && this.currentFolder?.path === '/'
```

Comment:
You could use `this.item === '/'` instead, right? Would like to avoid store usage when possible.